pom.xml

```diff
@@ -108,6 +108,14 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.mockito/mockito-core -->
```

Comment:
This dependency is unnecessary. Mockito is already provided by the parent

Reply:
I included this because [tests in travis could not be run due to a Mockito error](
https://travis-ci.org/NatLibFi/mauiservice/builds/546651120#L2271
): 

```
[ERROR] testCachedVocabulary(fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryTest)  Time elapsed: 0.606 s  <<< ERROR!

org.mockito.exceptions.base.MockitoException: 

Mockito cannot mock this class: class fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryImpl.

Mockito can only mock non-private & non-final classes.

If you're not sure why you're getting this error, please report to the mailing list.

Java               : 11

JVM vendor name    : Oracle Corporation

JVM vendor version : 11.0.2+9

JVM name           : OpenJDK 64-Bit Server VM

JVM version        : 11.0.2+9

JVM info           : mixed mode

OS name            : Linux

OS version         : 4.15.0-1028-gcp

Underlying exception : java.lang.UnsupportedOperationException: Cannot define class using reflection

	at fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryTest.setUp(SharedVocabularyMauiFilterFactoryTest.java:46)

Caused by: java.lang.UnsupportedOperationException: Cannot define class using reflection

	at fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryTest.setUp(SharedVocabularyMauiFilterFactoryTest.java:46)

Caused by: java.lang.IllegalStateException: Could not find sun.misc.Unsafe

	at fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryTest.setUp(SharedVocabularyMauiFilterFactoryTest.java:46)

Caused by: java.lang.NoSuchMethodException: sun.misc.Unsafe.defineClass(java.lang.String, [B, int, int, java.lang.ClassLoader, java.security.ProtectionDomain)

	at fi.nationallibrary.mauiservice.maui.SharedVocabularyMauiFilterFactoryTest.setUp(SharedVocabularyMauiFilterFactoryTest.java:46)
```

It looked like the problem described [here](https://code.scottshipp.com/2019/05/12/how-to-fix-mockbean-error-on-crudrepository-interface-in-a-spring-boot-app-using-java-11/) for which explicitly adding this dependency is mentioned as a solution. 

In the last [successfully ran test in Travis](https://travis-ci.org/NatLibFi/mauiservice/builds/434021672) 

```
$ java -Xmx32m -version
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
java version "1.8.0_181"
```

whereas in the more current, failing tests

```
$ java -Xmx32m -version
openjdk version "11.0.2" 2019-01-15
```

I'm not skilled in Java, so another solution for fixing erroring tests is appreciated!

Reply:
This is a good reason. Please update the comment in the pom to explain why it's there. "Newer version of mockito for Java 11 compatibility" or something like that.